src/Symfony/Component/HttpKernel/Controller/ControllerResolver.php

```diff
@@ -161,7 +161,7 @@ private function getControllerError($callable): string
             $availableMethods = $this->getClassMethodsWithoutMagicMethods($callable);
             $alternativeMsg = $availableMethods ? sprintf(' or use one of the available methods: "%s"', implode('", "', $availableMethods)) : '';
 
-            return sprintf('Controller class "%s" cannot be called without a method name. You need to implement "__invoke"%s.', \get_class($callable), $alternativeMsg);
+            return sprintf('Controller class "%s" cannot be called without a method name. You need to implement "__invoke"%s.', get_debug_type($callable), $alternativeMsg);
```

Comment:
three lines below you can use get_debug_type instead of gettype as well

Reply:
fixed thanks